src/ThrowableHandlerInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\ErrorHandler;
+
+use Throwable;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+
+interface ThrowableHandlerInterface
```

Comment:
`ErrorHandler` and `ThrowableHandler` are semantic is the same, but by fact is not. May by use name `ThrowableResponseFactory`? What do you think?

Reply:
First attempt was `ErrorResponseFactory` and it's close to `ThrowableResponseFactory`. Both sounds like _this factory creates response with error/throwable_. But this class handles throwable, uses request for it and generates response. So I call it like `RequestHandlerInterface` but for Throwable. It's better than ambiguous `Error/ThrowableResponseFactory`, but agree it' still not clear.

Reply:
`ThrowableRequestHandler` `ThrowableRequestHandlerInterface` maybe?

Reply:
Hm. But this class really create response for passed error/exception, isn't he?